docs/skyrim_platform/native.md

```diff
@@ -1,64 +1,118 @@
 # Native functions
 
-- Most types have a list of native functions, they are divided into static functions (`Native Global` in Papyrus) and methods (`Native`).
+Most types have a list of native functions, they are divided into static functions (`Native Global` in Papyrus) and methods (`Native`).
 
-- Static functions are called on the type:
+Static functions are called on the type:
 
-  ```typescript
-  let sunX = Game.getSunPositionX();
-  let pl = Game.getPlayer();
-  Game.forceFirstPerson();
-  ```
+```typescript
+let sunX = Game.getSunPositionX();
+let pl = Game.getPlayer();
+Game.forceFirstPerson();
+```
 
-- Methods are called on the object:
+Methods are called on the object:
 
-  ```typescript
-  let isPlayerInCombat = pl.isInCombat();
-  ```
+```typescript
+let isPlayerInCombat = pl.isInCombat();
+```
 
-- A list of original game types with documentation can be found here: https://www.creationkit.com/index.php?title=Category:Script_Objects
-- Calling functions from the original game is only available inside the `update` event handler (see below). If you try to do this in a different context, an exception will be thrown.
+A list of original game types with documentation can be found here: https://www.creationkit.com/index.php?title=Category:Script_Objects
+
+***WARNING***: Papyrus types and methods are only available inside [hooks][Hooks] or [new events][NewEvents] introduced by Skyrim Platform (except `tick`).
```

Comment:
Honestly, I think they are available only in papyrus event hook. In anim event hook not. Let's check it.